tests/CMakeLists.txt

```diff
@@ -29,7 +29,9 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-option(TESTS_USE_FORCED_PMEM "let tests force enable or force disable use of optimized flush in libpmemobj (to speed them up)" OFF)
+cmake_dependent_option(TESTS_USE_FORCED_PMEM
+	"let tests force enable or force disable use of optimized flush in libpmemobj (to speed them up)" OFF
+	"BUILD_LIBPMEMFILE_POSIX_TESTS;BUILD_LIBPMEMFILE_TESTS" OFF)
```

Comment:
Are you sure this is correct? The way I read the CMakeDependentOption documentation this makes TESTS_USE_FORCED_PMEM depend on *both* BUILD_LIBPMEMFILE_POSIX_TESTS and BUILD_LIBPMEMFILE_TESTS.

Reply:
I'm not sure if this is correct.

Reply:
Well, I guess disjunction is not supported by `cmake_dependent_option`, so I just reverted this part of the patch.